tests/dialects/test_bigquery.py

```diff
@@ -62,14 +67,14 @@ def test_bigquery(self):
         self.validate_all(
             "SELECT ARRAY_AGG(DISTINCT x IGNORE NULLS ORDER BY a, b DESC LIMIT 10) AS x",
             write={
-                "duckdb": "SELECT ARRAY_AGG(DISTINCT x ORDER BY a NULLS FIRST, b DESC LIMIT 10 IGNORE NULLS) AS x",
+                "duckdb": "SELECT ARRAY_AGG(DISTINCT x IGNORE NULLS ORDER BY a NULLS FIRST, b DESC LIMIT 10) AS x",
```

Comment:
I'd also include BQ in these tests, also the `"SELECT PERCENTILE_CONT(x, 0.5 RESPECT NULLS) OVER ()"` above with the two args